code/modules/research/rdconsole.dm

```diff
@@ -160,8 +160,7 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 		files.known_tech[T.id] = T
 	for(var/v in files.known_tech)
 		var/datum/tech/KT = files.known_tech[v]
-		if(KT.level < KT.max_level)
-			KT.level=KT.max_level
+		KT.level = 7
```

Comment:
is the max level of all tech 7 or lower?

Reply:
Scratch that, it's actually 8. So the button didn't even max those out in the first place, and if you wanted them to be at 8 easily, you had to scan the `/obj/item/research` debug item.

Reply:
So I guess my question is, if you were to hit the maxout research button with your current changes. Would it make everything available to build?

Reply:
I mean, I just switched it to 8 just now...